PatientWebApplication/UserMicroserviceApi/Controllers/PatientController.cs

```diff
@@ -23,36 +20,33 @@ public class PatientUserController : ControllerBase
         /// <value>Property <c>PatientService</c> represents the service used for handling business logic.</value>
         private PatientService PatientService { get; set; }
         private IWebHostEnvironment _env;
-        private RegularAppointmentService regularAppointmentService { get; set; }
-
 
-      public PatientUserController(IWebHostEnvironment env, MyDbContext dbContext)
-      {
-         PatientService = new PatientService(new PatientsRepository(dbContext), new EmailVerificationService(dbContext), new RegularAppointmentService(new AppointmentRepository(dbContext), new EmployeesScheduleRepository(dbContext), new DoctorService(new OperationRepository(dbContext), new AppointmentRepository(dbContext), new EmployeesScheduleRepository(dbContext), new DoctorRepository(dbContext)), new PatientsRepository(dbContext), new OperationService(new OperationRepository(dbContext))));
-         _env = env;
-      }
+        public PatientUserController(IWebHostEnvironment env, MyDbContext dbContext)
+        {
+            PatientService = new PatientService(new PatientsRepository(dbContext), new EmailVerificationService(dbContext));
+            _env = env;
+        }
 
-      /// <summary> This method determines if <c>PatientDto</c> provided <paramref name="dto"/> is valid for creating by calling <c>PatientValidator</c>
-      /// automatically and sends it to <c>PatientService</c>. </summary>  
-      /// <returns> if fields from <paramref name="dto"/> are not valid 400 Bad Request also if created feedback is not null 200 Ok else 404 Bad Request.</returns>
-      [HttpPost]
+        /// <summary> This method determines if <c>PatientDto</c> provided <paramref name="dto"/> is valid for creating by calling <c>PatientValidator</c>
+        /// automatically and sends it to <c>PatientService</c>. </summary>  
+        /// <returns> if fields from <paramref name="dto"/> are not valid 400 Bad Request also if created feedback is not null 200 Ok else 404 Bad Request.</returns>
+        [HttpPost]
+        [AllowAnonymous]
         public IActionResult Create(PatientDto patientDto)
         {
             if (PatientService.Create(patientDto) == null)
             {
                 return BadRequest();
             }
-
             return Ok();
-
-
         }
 
         /// <summary> This method calls <c>FeedbackService</c> to save image. </summary>
         /// <param name="file"><c>id</c> is File model that needs to be saved. 
         /// </param>
         /// <returns> If <paramref name="file"/> is not valid returns 400 Bad Request; if business logic is not valid, returns 404 Not Found, if path is successfully returned, returns 200 OK with path</returns>
         [HttpPost("image")]
+        [AllowAnonymous]
         public IActionResult SaveImg([FromForm] FileModel file)
         {
             string path = _env.WebRootPath;
```

Comment:
Variable 'path' directly forward in method ImageToSave(path, file) and after remove variable 'path'